### PR TITLE
Integrated CONTRIBUTING.rst content in Development section of docs.

### PR DESCRIPTION
**Pull request by andy-maier**
Ready for review and merge.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -121,12 +121,12 @@ The zhmcclient documentation is on RTD:
 .. _Documentation for version on Pypi: http://python-zhmcclient.readthedocs.io/en/stable/
 .. _Documentation for master branch in Git repo: http://python-zhmcclient.readthedocs.io/en/latest/
 
-Contributing
-============
+Development, test, and contributing
```
Comment (leopoldjuergen) [inline README.rst]: I would prefer 'testing' instead of 'test'.

Comment (andy-maier) [inline README.rst]: Done in commit 81d9f4f.

**Comment by andy-maier** [comment]
+1

**Comment by leopoldjuergen** [comment]
+1

**Pull request merged**